src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: Figure['shape'] = 'triangle';
+
+  color: Figure['color'];
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
```

Comment:
After moving color union to separate type you can add a type here rather than hardcode the required values
```suggestion
    color: Color,
```